src/WebJobs.Script.WebHost/Diagnostics/LinuxEventGenerator.cs

```diff
@@ -12,16 +12,23 @@ public abstract class LinuxEventGenerator : IEventGenerator
         public static readonly string EventTimestampFormat = "MM/dd/yyyy hh:mm:ss.fff tt";
 
         // These names should match the source file names for fluentd
-        public static readonly string FunctionsLogsCategory = "functionslogs";
+        public static readonly string FunctionsLogsCategory = "functionslogsv2";
```

Comment:
This isn't explicitly for Functions v2... it's v2 of the log template, right?

Reply:
Yes that is correct. v2 is just v2 version of that file. You bring up a good point that it may cause confusion in the future. hmmm.